### PR TITLE
[Security] Update Security Arabic Translations

### LINKED ISSUE #41028: Missing translations for Arabic (ar)
Hello,

There are some missing translation for Arabic (ar). These should be added in branch 4.4. 
See https://github.com/symfony/symfony/issues/38710 for details and [this page](https://symfony-translations.nyholm.tech/#pr) for an example.

These are the files that should be updated: 
- [src/Symfony/Component/Security/Core/Resources/translations/security.ar.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Security/Core/Resources/translations/security.ar.xlf)

NOTE: If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by mouadziani**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | Fix #41028
| License       | MIT
| Doc PR        | symfony/symfony-docs#... 

Related to #41028

### CODE REVIEW

File: src/Symfony/Component/Security/Core/Resources/translations/security.ar.xlf [hunk 1]
```diff
@@ -70,6 +70,14 @@
                 <source>Invalid or expired login link.</source>
                 <target>رابط تسجيل الدخول غير صالح أو منتهي الصلاحية.</target>
             </trans-unit>
+            <trans-unit id="19">
+                <source>Too many failed login attempts, please try again in %minutes% minute.</source>
+                <target>عدد كبير جدا من محاولات الدخول الفاشلة، يرجى اعادة المحاولة بعد %minutes% دقيقة.</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Security/Core/Resources/translations/security.ar.xlf:75]: Does this say "minute", ie singular? 

trans-unit id="20" should say "minutes" in plural.

**Review by Nyholm — COMMENTED** [review]
Awesome. Thank you. I just have one question to confirm.

File: src/Symfony/Component/Security/Core/Resources/translations/security.ar.xlf (hunk 1 above)
Comment (mouadziani) [reply src/Symfony/Component/Security/Core/Resources/translations/security.ar.xlf:75]: In Arabic we can say (دقيقة) for both singular and plural

Comment (Nyholm) [reply src/Symfony/Component/Security/Core/Resources/translations/security.ar.xlf:75]: Cool. Thank you for confirming.

**Review by Nyholm — APPROVED** [review]

**Comment by derrabus** [comment]
Thank you @MouadZIANI.

**Comment by MouadZIANI** [comment]
LGTM

**Pull request merged**